libkbfs/keybase_service_base.go

```diff
@@ -414,7 +441,10 @@ func (k *KeybaseServiceBase) LoadUserPlusKeys(ctx context.Context,
 // KeybaseServiceBase.
 func (k *KeybaseServiceBase) LoadTeamPlusKeys(
 	ctx context.Context, tid keybase1.TeamID) (TeamInfo, error) {
-	// No caching until the invalidations are ready.
+	cachedTeamInfo := k.getCachedTeamInfo(tid)
+	if cachedTeamInfo.Name != libkb.NormalizedUsername("") {
```

Comment:
Sorry, I don't follow the question.  You mean if `tid` isn't a valid team ID, we should remember that and save ourselves the cost of checking it later?  I'm confused, because this line of code has nothing to do with team IDs.  It's only checking whether the cache is populated for this team ID or not.

(This works the same way as the user cache.  Though it's probably better to actually delete stuff from the cache, rather than setting them to a zero-value, not sure why we don't do that.)